### PR TITLE
V0.5 changelog scratchpad

### PR DESCRIPTION
**Pull request by geoffthemedio**
WIP updating as I go: https://github.com/freeorion/freeorion/blob/v0.5_changelog_scratchpad/changelog_scratchpad

### CODE REVIEW

**Comment by geoffthemedio** [comment]
@Oberlus @agrrr3 Feedback requested on whether anything put into the change list so far actually duplicates something that was in v0.4.10, v0.4.10.1, or v0.4.10.2

**Comment by Oberlus** [comment]
Not sure if this is what you need:

The following commits listed in this scrathpad were later cherry-picked for v0.4.10.*:

In the summary at the end:
- Made Chato BAD_POPULATION
- Delayed effects of Experimentor outpost 50 turns

In the raw listing above it:
- Replace Arc Disruptor icons with online-edited versions
- Fix Bio-Adaptive hull regeneration description.

**Comment by geoffthemedio** [comment]
> Not sure if this is what you need:

That sort of info is useful.

**Comment by o01eg** [comment]
Maybe boost and python minimum version changes should be mentioned in `Technical / Internal` - `Dependencies and Versions`, not `Key changes`?

Also I suppose add remark about experimental Godot client and Android support.

In `Technical / Internal` also add that game rules and techs use Python for definition.

**Comment by agrrr3** [comment]
maybe say that stability was formerly known as happiness; there also some minor gripes; but i cant directly comment on the code lines..
"and damaged adjusted accordingly" -> "and damage adjusted accordingly"

i think the flux hulls have their own hull line and are in no way considered robotic anymore

the close combat weapons should be noted (flux lance and monster weapons)

also i am not sure fixes should go into the release notes if the bug did not exist in 0.4.10 (but hard to say just looking at the fix descriptions... probably not worth the effort finding out -maybe simply group fixes together )

"Fixed Sly refueling detection of systems with gas giants" -> "Fixed Sly refueling in systems with gas giants" (i think; maybe that feature got added in 0.4.10+)

also -Stringtables "In many cases show actual calculated in-game values (named ValueRefs)"

**Comment by geoffthemedio** [comment]
> i am not sure fixes should go into the release notes if the bug did not exist in 0.4.10

I skip over many commit log entries that are not relevant in that way, though it's difficult to know in some/many cases whether an issue was in v0.4.10 or introduced by changes since then.

**Comment by Grummel7** [comment]
```
#### AI

- AI manages influence and adopts policies
- Devalued Gaia special as Sly
- Corrected colony pod turns_to_build when Lifecycle Manipulation is available
- Fixed issue with weapons that have 0 shots
- Made AI tech enqueue order more consistent between different machines
```

Since I did most of the work on the AI, here is how I would summarise my modifications:
- AI adopts policies
- AI manages influence and stability
- AI builds Regional Administrators and Nearly Universal Translators
- Improved invasion handling
- AI no longer uses protection mode for military purposes
- Many refactorings, small improvements and adaptations to changed game rules.

Also many bug fixes, although I am not sure how many of those bug existed in 4.10. I am pretty sure, invasion was already broken in 4.10 and even before that, though (AI always had lots of troop ships idling around).

Two things I found before my time:
- Let AI not colonize in unexplored systems
- Add option to allow human player to take over any AI (#3357)

**Comment by geoffthemedio** [comment]
@Cjkjvfnby your input on what to include for AI changes would be helpful...

File: changelog_scratchpad [hunk 1]
```diff
@@ -23755,15 +23754,15 @@ Author: Oberlus <sicofonte@gmail.com>
...
-    - Added policies
+    - Added numerous policies that empires can adopt
 
 - Ships
     - Ship structure and damage adjusted for 4 bout combats
-    - Added ROBOTIC_HULL tag to Space Flux Bubble hull
-    - Fixed Sly refueling detection of systems with gas giants
-    - Various cost rebalancing
-    - Apply ship weapon damage rule scaling to species weapon bonuses
+    - Added the Flux Lance weapon that is "short range" any only attacks bout 3 (or 2 if the Charge policy is adopted)
```
Comment (agrrr3) [inline changelog_scratchpad]: it attacks starting from bout 3

File: changelog_scratchpad [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+
+#### AI
+
+- AI adopts policies
+- AI Manages influence and stability
+- AI builds Regional Administrations and Nearly Universal Translators
+- Improved invasion handling
+- AI no longer uses protection focus for military purposes
+- Many refactorings and adaptations for changed rules and content
+- Devalued Gaia special as Sly
```
Comment (Grummel7) [inline changelog_scratchpad:284]: I would remove that line. I've re-written the entire planet evaluation code, which includes the fact that for Gaia there are three species who cannot use it (Exobots, Sly and Celest-something), but players never knew why the AI settles a specific planet.

Apart from that, the list should be fine.

File: changelog_scratchpad [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+    [AI] Fix _get_queued_base_troopers (#3791)
+    [AI] Fixes for drop invasion (#3783)
+    [AI] Improve AI invasion handling (#3754)
+    [AI] Ensure colony_buildes are initialized before usage (#3744)
+    [AI] Consider researching space flux hulls if you dont research organics
+
+
+    [AI] Building refactoring (#3808)
+    - Scanning Facility: Don't build it where it is disliked
+    - Gas Giant Generator: Completely new logic
```
Comment (geoffthemedio) [inline changelog_scratchpad:36]: @Grummel7 I wanted your input on this list of [AI] items

**Comment by geoffthemedio** [comment]
https://github.com/freeorion/freeorion/pull/4338

**Pull request closed without merging**